dali/kernels/imgproc/convolution/cutlass/utility.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I wonder if convolution kernels should live in the imgproc as we can think about reusing them in audio as well. But this is rather a side note.

Reply:
I can move all of them 1 up as a separate PR, but it's rather insignificant change.